test/proxy_SUITE.erl

```diff
@@ -402,11 +411,20 @@ limit_exceeded(_Config) ->
     ok = libp2p_swarm:stop(ClientSwarm2),
 
     %% check we didn't leak any sockets here
-    ok = test_util:wait_until(fun() ->
-                                      [{_ID, Info}] = ranch:info(),
-                                      0 == proplists:get_value(active_connections, Info) andalso
-                                      0 == proplists:get_value(all_connections, Info)
-                              end),
+    ok = test_util:wait_until(
+        fun() ->
+            lists:foldl(
+                fun({_ID, _Info}, false) ->
+                    false;
+                ({_ID, Info}, _Acc) ->
+                    0 == proplists:get_value(active_connections, Info) andalso
```

Comment:
Use assertions here

Reply:
I can't use them here cause it's in the wait_until